packages/api/__tests__/api/private/comment.graphql

```diff
@@ -0,0 +1,17 @@
+fragment MutationComment on Comment {
+  itemId
```

Comment:
Here also `itemID` or is there a specific reason that you need to call it `Id` ?

Reply:
fixed